### PR TITLE
Improve readme structure

### PR DESCRIPTION
**Pull request by cadamini**
**Description**
This pull request (hopefully) improves the readme. Let me know if I was totally wrong :)

**Issue**
none

**Testing Instructions**
Visit https://github.com/cadamini/frontend/tree/patch-1

**Additional Notes**
none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,49 @@
...
 
-This is the official frontend of Cryptic.  
-https://play.cryptic-game.net/
+This is the official [Cryptic](https://play.cryptic-game.net/) frontend ([available at docker-hub](https://hub.docker.com/r/crypticcp/frontend/)). It has been created with the **Angular** framework using _TypeScript_, _HTML_ and [SASS](http://sass-lang.com/guide).  
 
-# Advice for Developers
+## Set up a development environment
 
-This web application is created with the **Angular** framework. The Framework uses the languages _TypeScript_ (JavaScript with typing), _HTML_ and _SCSS_. A fundamental understanding of these languages is important to understand the code. _SCSS_ or [_Sass_](http://sass-lang.com/guide) is a CSS preprocessor. You can find a tutorial [here](https://www.toptal.com/sass/theming-scss-tutorial).
+NodeJS and NPM are required to compile and start of the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
```
Comment (cadamini) [inline README.md:9]:
```suggestion
NodeJS and NPM are required to compile and start the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
```

File: README.md [hunk 2]
```diff
@@ -1,97 +1,49 @@
...
 
-The framework also uses a **CLI** (_Comand Line Interface_). It helps creating and managing Angular projects fast.
+[Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. Visual Studio Code (VSCode) provides a build-in debugger and a _git_ client.  
 
-## Editor
+To open the repository in **VSCode**:
```
Comment (cadamini) [inline README.md:13]: I have used a wrong intro sentence

File: README.md [hunk 3]
```diff
@@ -1,97 +1,49 @@
...
 
-### Further help
+To check browser-compatibility, use [Can I use](https://caniuse.com/). You should test your code at minimum with one WebKit browser (Chrome, Chromium, Safari, or Opera) and one non-WebKit browser (Firefox, Internet Explorer, or Edge). [CSS tricks](https://css-tricks.com) delivers tips for CSS/SCSS (e.g. centering, flex-box, grid etc.).
 
-The [Angular documentation](https://angular.io/docs) is very helpful. `ng --help` or the [Angular CLI documentation](https://github.com/angular/angular-cli/wiki) can help with the _CLI_
+## Debugging
 
-### Docker-Hub
-This frontend is online on docker-hub (https://hub.docker.com/r/crypticcp/frontend/).
+VSCode comes with a build-in, ready-to-use [debugger](https://code.visualstudio.com/Docs/editor/debugging). Run  the task `run`, switch to the debug section, and start `Run`. **Important:** For the debugger the extension, _Debugger for Chrome_ must be installed.
```
Comment (cadamini) [inline README.md:49]: The task part is still vaque, and can we add a link to the extension? 

```suggestion
VSCode comes with a build-in, ready-to-use [debugger](https://code.visualstudio.com/Docs/editor/debugging). To use it, run  the task `run`, switch to the debug section, and start `Run`. The debugger requires the  _Debugger for Chrome_  Chrome extension. 
```

File: README.md (hunk 2 above)
Comment (cadamini) [reply README.md:13]:
```suggestion
If you use VSCode**:  
```

File: README.md [hunk 4]
```diff
@@ -1,97 +1,49 @@
...
+## Set up a development environment
 
-This web application is created with the **Angular** framework. The Framework uses the languages _TypeScript_ (JavaScript with typing), _HTML_ and _SCSS_. A fundamental understanding of these languages is important to understand the code. _SCSS_ or [_Sass_](http://sass-lang.com/guide) is a CSS preprocessor. You can find a tutorial [here](https://www.toptal.com/sass/theming-scss-tutorial).
+NodeJS and NPM are required to compile and start the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
 
-The framework also uses a **CLI** (_Comand Line Interface_). It helps creating and managing Angular projects fast.
+[Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. Visual Studio Code (VSCode) provides a build-in debugger and a _git_ client.  
 
-## Editor
+If you use VSCode**:
```
Comment (cadamini) [inline README.md:13]:
```suggestion
If you use **VSCode**:  
```

File: README.md [hunk 5]
```diff
@@ -1,97 +1,49 @@
...
 
-# Advice for Developers
+## Set up a development environment
 
-This web application is created with the **Angular** framework. The Framework uses the languages _TypeScript_ (JavaScript with typing), _HTML_ and _SCSS_. A fundamental understanding of these languages is important to understand the code. _SCSS_ or [_Sass_](http://sass-lang.com/guide) is a CSS preprocessor. You can find a tutorial [here](https://www.toptal.com/sass/theming-scss-tutorial).
+NodeJS and NPM are required to compile and start the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
 
-The framework also uses a **CLI** (_Comand Line Interface_). It helps creating and managing Angular projects fast.
+[Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. Visual Studio Code (VSCode) provides a build-in debugger and a _git_ client.
```
Comment (MarcelCoding) [inline README.md:11]: Change this to a list and add IntelliJ Ultimate/Webstorm (or with a student license)

File: README.md [hunk 6]
```diff
@@ -1,97 +1,49 @@
...
+3. Run `install` to install all required dependencies. On Linux/Mac OSX you must type in your password.
+4. Execute the task `run`, which starts an development server listening on [http://localhost:4200](http://localhost:4200).
 
-## Setting up an Development Environment
+If you use the **CLI**: 
 
-For compiling and starting of the application **NodeJS** and **NPM** are needed. Downloads and installation advice can be found on [https://nodejs.org/de/](https://nodejs.org/de/).
+1. Open the repository in the terminal. 
+2. Run `npm install`. Brew a fresh coffee, because this process can take a bit time.
+3. Run `ng serve` or `npm start` to start a development server listening on [http://localhost:4200](http://localhost:4200), .
```
Comment (MarcelCoding) [inline README.md:24]: Add IntelliJ/Webstorm:
1. File -> New -> Project from Version Control
2. paste git url
3. twice `SHIFT` -> `npm install`
4. Top Right `Angular CLI Server` green play button

File: README.md [hunk 7]
```diff
@@ -1,97 +1,49 @@
...
-Before you use very new or old things in your code it's recommendet to use "[Can I use](https://caniuse.com/)". With this tool you can test the compatibility of your code.
+- `ng build`: compiles and builds the whole project. The result will be stored in `dist/`. For production mode use the `--prod` argument.
+- `ng test`: tests your code with [Karma](https://karma-runner.github.io).
+- `ng e2e`: runs end-to-end tests with [Protractor](http://www.protractortest.org/).
 
-[CSS tricks](https://css-tricks.com) delivers tipps for CSS/SCSS (e.g. centering, flex-box, grid etc.).
+## Check browser compatibility
 
-### Further help
+To check browser-compatibility, use [Can I use](https://caniuse.com/). You should test your code at minimum with one WebKit browser (Chrome, Chromium, Safari, or Opera) and one non-WebKit browser (Firefox, Internet Explorer, or Edge). [CSS tricks](https://css-tricks.com) delivers tips for CSS/SCSS (e.g. centering, flex-box, grid etc.).
```
Comment (MarcelCoding) [inline README.md:45]: Angular dropped support for IE & move Edge to WebKit

Comment (MarcelCoding) [reply README.md:45]: Put the css help in a seperate line. (`<br>` - break)

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (cadamini) [reply README.md:11]: Idk the tool, was it invluded before? Need to do some research. Can you add a suggestion.using the button to the top left? @MarcelCoding

File: README.md (hunk 7 above)
Comment (cadamini) [reply README.md:45]:
```suggestion
To check browser-compatibility, use [Can I use](https://caniuse.com/). You should test your code at minimum with one WebKit browser (Chrome, Chromium, Safari, or Opera) and one non-WebKit browser (Firefox or Microsoft Edge). 

Note: [CSS tricks](https://css-tricks.com) delivers tips for CSS/SCSS (e.g. centering, flex-box, grid etc.).
```

File: README.md (hunk 5 above)
Comment (cadamini) [reply README.md:11]:
```suggestion
- [Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. It provides a build-in debugger and a _git_ client.  
- IntelliJ Ultimate/Webstorm (paid or student license) 
```

**Comment by cadamini** [comment]
@MarcelCoding ready but see last commit. Is this correct?

**Comment by MarcelCoding** [comment]
@cadamini I edited the code in your comment. (welcome screen) Could you integrate that?

File: README.md [hunk 8]
```diff
@@ -1,97 +1,59 @@
...
-For compiling and starting of the application **NodeJS** and **NPM** are needed. Downloads and installation advice can be found on [https://nodejs.org/de/](https://nodejs.org/de/).
+1. Open the repository in the terminal. 
+2. Run `npm install`. Brew a fresh coffee, because this process can take a bit time.
+3. Run `ng serve` or `npm start` to start a development server listening on [http://localhost:4200](http://localhost:4200). 
 
-### With VSCode
+If you use **IntelliJ/Webstorm**:
 
-After the installation of _Node_ an _NPM_ open the repository in VSCode. Press _Ctrl+Shift+P / CMD+Shift+P_ to open the command palette. Type `Task: Run Task` to enter task mode. The task `install` installs all dependencies needed.
+1. Click File -> New -> Project from Version Control
```
Comment (cadamini) [inline README.md:29]:
```suggestion
1. Click File -> New -> Project from Version Control. 
```

Comment (cadamini) [inline README.md:29]:
```suggestion
1. Click File -> New -> Project from Version Control (Welcome Screen: `Get from VSC`)
```

File: README.md [hunk 9]
```diff
@@ -1,97 +1,59 @@
...
-After this you can execute the task `run`. It starts an development server listening on [http://localhost:4200](http://localhost:4200).
-
-### With the CLI
-
-After the installation process open the repository in the terminal and run `npm install`. Make yourself a coffee, because this process can take a bit time.
-
-To start a development server on [http://localhost:4200](http://localhost:4200) run `ng serve`. This is also possible with `npm start`.
-
-## VSCode
+Note: You can reate a new component using the command `ng generate component component-name` (short: `ng g c component-name`).
```
Comment (cadamini) [inline README.md:34]: Typo

Comment (cadamini) [reply README.md:34]:
```suggestion
Note: Create a new component with the command `ng generate component component-name` (short: `ng g c component-name`).
```

File: README.md [hunk 10]
```diff
@@ -1,97 +1,59 @@
...
 
-### Further help
+Note: [CSS tricks](https://css-tricks.com) delivers tips for CSS/SCSS (e.g. centering, flex-box, grid etc.).
 
-The [Angular documentation](https://angular.io/docs) is very helpful. `ng --help` or the [Angular CLI documentation](https://github.com/angular/angular-cli/wiki) can help with the _CLI_
+## Debugging
 
-### Docker-Hub
-This frontend is online on docker-hub (https://hub.docker.com/r/crypticcp/frontend/).
+VSCode comes with a build-in, ready-to-use [debugger](https://code.visualstudio.com/Docs/editor/debugging)  that requires the  _Debugger for Chrome_  browser extension. To use it, run  the task `run`, switch to the debug section, and start `Run`.
```
Comment (cadamini) [inline README.md:59]:
```suggestion
VSCode comes with a build-in, ready-to-use [debugger](https://code.visualstudio.com/Docs/editor/debugging)  that requires the  _Debugger for Chrome_ browser extension. To use it, run  the task `run`, switch to the debug section, and start `Run`.
```

File: README.md [hunk 11]
```diff
@@ -1,97 +1,59 @@
...
 
-# Advice for Developers
+## Set up a development environment
 
-This web application is created with the **Angular** framework. The Framework uses the languages _TypeScript_ (JavaScript with typing), _HTML_ and _SCSS_. A fundamental understanding of these languages is important to understand the code. _SCSS_ or [_Sass_](http://sass-lang.com/guide) is a CSS preprocessor. You can find a tutorial [here](https://www.toptal.com/sass/theming-scss-tutorial).
+NodeJS and NPM are required to compile and start the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
 
-The framework also uses a **CLI** (_Comand Line Interface_). It helps creating and managing Angular projects fast.
+- [Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. It provides a build-in debugger and a _git_ client.
```
Comment (cadamini) [inline README.md:11]:
```suggestion
- [Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. It provides a build-in debugger (see [debugging] (#debugging)) and integrates a _git_ client.  
```

**Review by MarcelCoding — DISMISSED** [review]
LGTM

**Comment by cadamini** [comment]
@MarcelCoding one more approval, the review from @MaxiHuHe04 is pending for a while. Can you pull in someone else?

File: README.md [hunk 12]
```diff
@@ -1,97 +1,59 @@
-# official frontend
+# Official Cryptic Frontend
 
 [![Build Status](https://travis-ci.org/cryptic-game/frontend.svg?branch=master)](https://travis-ci.org/cryptic-game/frontend)
 
-This is the official frontend of Cryptic.  
-https://play.cryptic-game.net/
+This is the official [Cryptic](https://play.cryptic-game.net/) frontend ([available at docker-hub](https://hub.docker.com/r/crypticcp/frontend/)). It has been created with the **Angular** framework using _TypeScript_, _HTML_ and [SASS](http://sass-lang.com/guide).
```
Comment (Defelo) [inline README.md:5]:
```suggestion
This is the official [Cryptic frontend](https://play.cryptic-game.net/) (available on [Docker Hub](https://hub.docker.com/r/crypticcp/frontend/)). It has been created with the **Angular** framework using _TypeScript_, _HTML_ and [SASS](http://sass-lang.com/guide).  
```

File: README.md [hunk 13]
```diff
@@ -1,97 +1,59 @@
...
 
-# Advice for Developers
+## Set up a development environment
 
-This web application is created with the **Angular** framework. The Framework uses the languages _TypeScript_ (JavaScript with typing), _HTML_ and _SCSS_. A fundamental understanding of these languages is important to understand the code. _SCSS_ or [_Sass_](http://sass-lang.com/guide) is a CSS preprocessor. You can find a tutorial [here](https://www.toptal.com/sass/theming-scss-tutorial).
+NodeJS and NPM are required to compile and start the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
 
-The framework also uses a **CLI** (_Comand Line Interface_). It helps creating and managing Angular projects fast.
+- [Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. It provides a build-in debugger (see [debugging](#debugging)) and integrates a _git_ client.
```
Comment (Defelo) [inline README.md:11]:
```suggestion
- [Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. It provides a built-in debugger (see [debugging](#debugging)) and integrates a _git_ client.  
```

File: README.md [hunk 14]
```diff
@@ -1,97 +1,59 @@
...
 
-# Advice for Developers
+## Set up a development environment
 
-This web application is created with the **Angular** framework. The Framework uses the languages _TypeScript_ (JavaScript with typing), _HTML_ and _SCSS_. A fundamental understanding of these languages is important to understand the code. _SCSS_ or [_Sass_](http://sass-lang.com/guide) is a CSS preprocessor. You can find a tutorial [here](https://www.toptal.com/sass/theming-scss-tutorial).
+NodeJS and NPM are required to compile and start the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
 
-The framework also uses a **CLI** (_Comand Line Interface_). It helps creating and managing Angular projects fast.
+- [Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. It provides a build-in debugger (see [debugging](#debugging)) and integrates a _git_ client.  
+- IntelliJ Ultimate/Webstorm (paid or student license)
```
Comment (Defelo) [inline README.md:12]:
```suggestion
- [WebStorm](https://www.jetbrains.com/webstorm/) (paid or student license) 
```

File: README.md [hunk 15]
```diff
@@ -1,97 +1,59 @@
...
+2. Type `Task: Run Task` to enter task mode. 
+3. Run `install` to install all required dependencies. On Linux/Mac OSX you must type in your password.
+4. Execute the task `run`, which starts an development server listening on [http://localhost:4200](http://localhost:4200).
 
-## Setting up an Development Environment
+If you use the **CLI**: 
 
-For compiling and starting of the application **NodeJS** and **NPM** are needed. Downloads and installation advice can be found on [https://nodejs.org/de/](https://nodejs.org/de/).
+1. Open the repository in the terminal. 
+2. Run `npm install`. Brew a fresh coffee, because this process can take a bit time.
```
Comment (Defelo) [inline README.md:24]:
```suggestion
2. Run `npm install`. Brew a fresh coffee, because this process can take some time.
```

File: README.md [hunk 16]
```diff
@@ -1,97 +1,59 @@
...
+3. Run `ng serve` or `npm start` to start a development server listening on [http://localhost:4200](http://localhost:4200). 
 
-### With VSCode
+If you use **IntelliJ/Webstorm**:
```
Comment (Defelo) [inline README.md:27]:
```suggestion
If you use **WebStorm**:
```

File: README.md [hunk 17]
```diff
@@ -1,97 +1,59 @@
...
 
-### Further help
+Note: [CSS tricks](https://css-tricks.com) delivers tips for CSS/SCSS (e.g. centering, flex-box, grid etc.).
 
-The [Angular documentation](https://angular.io/docs) is very helpful. `ng --help` or the [Angular CLI documentation](https://github.com/angular/angular-cli/wiki) can help with the _CLI_
+## Debugging
 
-### Docker-Hub
-This frontend is online on docker-hub (https://hub.docker.com/r/crypticcp/frontend/).
+VSCode comes with a build-in, ready-to-use [debugger](https://code.visualstudio.com/Docs/editor/debugging)  that requires the  _Debugger for Chrome_ browser extension. To use it, run  the task `run`, switch to the debug section, and start `Run`.
```
Comment (Defelo) [inline README.md:59]:
```suggestion
VSCode comes with a built-in, ready-to-use [debugger](https://code.visualstudio.com/Docs/editor/debugging)  that requires the  _Debugger for Chrome_ browser extension. To use it, run  the task `run`, switch to the debug section, and start `Run`.
```

**Review by Defelo — CHANGES_REQUESTED** [review]

**Review by Defelo — APPROVED** [review]

**Review by MarcelCoding — APPROVED** [review]

**Pull request merged**